<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="15" passed="13" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-05-06T20:03:59 IST" name="Suite" finished-at="2025-05-06T20:04:28 IST" duration-ms="29592">
    <groups>
    </groups>
    <test started-at="2025-05-06T20:03:59 IST" name="Test" finished-at="2025-05-06T20:04:28 IST" duration-ms="29592">
      <class name="api.testcases.StoreTest">
        <test-method is-config="true" signature="generateStoreTestData()[pri:0, instance:api.testcases.StoreTest@23e84203]" started-at="2025-05-06T20:04:19 IST" name="generateStoreTestData" finished-at="2025-05-06T20:04:19 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateStoreTestData -->
        <test-method signature="testCreateAnOrderData()[pri:1, instance:api.testcases.StoreTest@23e84203]" started-at="2025-05-06T20:04:19 IST" name="testCreateAnOrderData" description="Place an order for Pet" finished-at="2025-05-06T20:04:20 IST" duration-ms="1176" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateAnOrderData -->
        <test-method signature="testGetAnOrderData()[pri:2, instance:api.testcases.StoreTest@23e84203]" started-at="2025-05-06T20:04:20 IST" name="testGetAnOrderData" description="Find purchase order by ID" finished-at="2025-05-06T20:04:21 IST" depends-on-methods="api.testcases.StoreTest.testCreateAnOrderData" duration-ms="1210" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetAnOrderData -->
        <test-method signature="testDeleteAnOrderData()[pri:3, instance:api.testcases.StoreTest@23e84203]" started-at="2025-05-06T20:04:21 IST" name="testDeleteAnOrderData" description="Delete purchase order by ID" finished-at="2025-05-06T20:04:22 IST" depends-on-methods="api.testcases.StoreTest.testCreateAnOrderData" duration-ms="1159" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteAnOrderData -->
      </class> <!-- api.testcases.StoreTest -->
      <class name="api.testcases.UserTest">
        <test-method is-config="true" signature="generateTestData()[pri:0, instance:api.testcases.UserTest@17fc391b]" started-at="2025-05-06T20:03:59 IST" name="generateTestData" finished-at="2025-05-06T20:03:59 IST" duration-ms="583" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateTestData -->
        <test-method signature="testCreateUserData()[pri:1, instance:api.testcases.UserTest@17fc391b]" started-at="2025-05-06T20:03:59 IST" name="testCreateUserData" description="End to end  Operations about user" finished-at="2025-05-06T20:04:03 IST" duration-ms="3064" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateUserData -->
        <test-method signature="testGetUserData()[pri:2, instance:api.testcases.UserTest@17fc391b]" started-at="2025-05-06T20:04:03 IST" name="testGetUserData" description="End to end  Get user by user name" finished-at="2025-05-06T20:04:04 IST" depends-on-methods="api.testcases.UserTest.testCreateUserData" duration-ms="1111" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetUserData -->
        <test-method signature="testUpdateUserData()[pri:3, instance:api.testcases.UserTest@17fc391b]" started-at="2025-05-06T20:04:04 IST" name="testUpdateUserData" description="End to end  Updated user by user name" finished-at="2025-05-06T20:04:06 IST" depends-on-methods="api.testcases.UserTest.testCreateUserData" duration-ms="2182" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdateUserData -->
        <test-method signature="testDeleteUserData()[pri:4, instance:api.testcases.UserTest@17fc391b]" started-at="2025-05-06T20:04:06 IST" name="testDeleteUserData" description="End to end  Delete user by user name" finished-at="2025-05-06T20:04:07 IST" depends-on-methods="api.testcases.UserTest.testCreateUserData" duration-ms="1070" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserData -->
      </class> <!-- api.testcases.UserTest -->
      <class name="api.testcases.PetTest">
        <test-method is-config="true" signature="generatePetTestData()[pri:0, instance:api.testcases.PetTest@792b749c]" started-at="2025-05-06T20:04:07 IST" name="generatePetTestData" finished-at="2025-05-06T20:04:07 IST" duration-ms="36" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generatePetTestData -->
        <test-method signature="testCreatePetData()[pri:1, instance:api.testcases.PetTest@792b749c]" started-at="2025-05-06T20:04:07 IST" name="testCreatePetData" description="End to end Everything about your Pets" finished-at="2025-05-06T20:04:09 IST" duration-ms="1965" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreatePetData -->
        <test-method signature="testGetPetData()[pri:2, instance:api.testcases.PetTest@792b749c]" started-at="2025-05-06T20:04:09 IST" name="testGetPetData" description="End to end  Get user by Pet Id" finished-at="2025-05-06T20:04:10 IST" depends-on-methods="api.testcases.PetTest.testCreatePetData" duration-ms="1078" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetPetData -->
        <test-method signature="testUpdatePetData()[pri:3, instance:api.testcases.PetTest@792b749c]" started-at="2025-05-06T20:04:10 IST" name="testUpdatePetData" description="End to end Updated existing pet" finished-at="2025-05-06T20:04:12 IST" depends-on-methods="api.testcases.PetTest.testCreatePetData" duration-ms="2159" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdatePetData -->
        <test-method signature="testDeletePetData()[pri:4, instance:api.testcases.PetTest@792b749c]" started-at="2025-05-06T20:04:12 IST" name="testDeletePetData" description="End to end  Delete user by petId" finished-at="2025-05-06T20:04:19 IST" depends-on-methods="api.testcases.PetTest.testCreatePetData" duration-ms="6468" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [405]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at api.testcases.PetTest.testDeletePetData(PetTest.java:173)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeletePetData -->
      </class> <!-- api.testcases.PetTest -->
      <class name="api.testcases.UserTest2">
        <test-method is-config="true" signature="generateTestData()[pri:0, instance:api.testcases.UserTest2@19932c16]" started-at="2025-05-06T20:04:22 IST" name="generateTestData" finished-at="2025-05-06T20:04:22 IST" duration-ms="19" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateTestData -->
        <test-method signature="testCreateUserData()[pri:1, instance:api.testcases.UserTest2@19932c16]" started-at="2025-05-06T20:04:22 IST" name="testCreateUserData" description="End to end  Operations about user" finished-at="2025-05-06T20:04:23 IST" duration-ms="1209" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateUserData -->
        <test-method signature="testGetUserData()[pri:2, instance:api.testcases.UserTest2@19932c16]" started-at="2025-05-06T20:04:23 IST" name="testGetUserData" description="End to end  Operations about user" finished-at="2025-05-06T20:04:25 IST" duration-ms="1230" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetUserData -->
        <test-method signature="testUpdateUserData()[pri:2, instance:api.testcases.UserTest2@19932c16]" started-at="2025-05-06T20:04:25 IST" name="testUpdateUserData" description="End to end  Operations about user" finished-at="2025-05-06T20:04:27 IST" duration-ms="2426" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdateUserData -->
        <test-method signature="testDeleteUserData()[pri:4, instance:api.testcases.UserTest2@19932c16]" started-at="2025-05-06T20:04:27 IST" name="testDeleteUserData" description="End to end  Operations about user" finished-at="2025-05-06T20:04:28 IST" duration-ms="1104" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at api.testcases.UserTest2.testDeleteUserData(UserTest2.java:123)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserData -->
      </class> <!-- api.testcases.UserTest2 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
